### ISSUE TITLE
Cannot move Client class def/impl to separate files.

### ISSUE THREAD

**Issue by imalyavskiy** [OpenVPN/openvpn3#62]
Hello I would like to use several cpp files to build this and dependent project. But currently cannot do this because if to include opnvpn headers into at least two cpp files the linker angries with a lot of "already defined in cli.obj" errors. What is the reason for the headers to be designed in this way or is there any way to use the headers in desired manner without changing them? 

Thanks in advance.

PS: Working on Win 10.

**Comment by schwabe** [comment]
Duplicate of #48. See also the openvpn3-linux client as example how to use multiple cpp files and including openvpn3 only once.

**Comment by imalyavskiy** [comment]
Thanks a lot!
